MIGRATION.md

```diff
@@ -34,43 +34,94 @@ This new path points to the `dist` directory, ensuring that you're importing the
 ## From version 3.0.0 to 4.0.0
 
 ### Brand Colors
-
-### Brand Colors Brand Evolution
+- Grey updates
+- grey000 added
+- grey025 added
+- grey050 added
+- grey030 removed
+- grey040 removed
+- grey750 removed
+- grey1000 added
```

Comment:
In the final migration documentation, it would be beneficial to include specific code changes.  Added 
JS
Added
```
brandColor.grey000
brandColor.grey025
```
Removed
```
brandColor.grey030
```

CSS
Added
```
--brand-colors-grey-grey000 
--brand-colors-grey-grey025
```
Removed
```
--brand-colors-grey-grey030
```


This detail will aid engineers who are searching for removed tokens in the codebase. cc @garrettbear

Reply:
do you want me to update this to reflect that?

Reply:
tho the colors will be the same for both changes, the difference is syntax. curious if theres a better way to list this instead of duplicating it